Dotnet.Homeworks.MainProject/Configuration/RabbitMqConfig.cs

```diff
@@ -0,0 +1,15 @@
+namespace Dotnet.Homeworks.MainProject.Configuration;
+
+public class RabbitMqConfig
+{
+    public RabbitMqConfig(string username, string password, string hostname)
+    {
+        Username = username;
+        Password = password;
+        Hostname = hostname;
+    }
+
+    public string Username { get; private set; }
```

Comment:
в таком конфиге нет локальных методов которые меняют параметры. Достаточно оставить init вместо set
```suggestion
    public string Username { get; init; }
```

Reply:
создавать конструктор - было ошибкой. этот конфиг должен байндиться из конфлигурации автоматически. убрал конструктор вообще и заменил private set на set